app/views/unattended/provisioning_templates/snippet/kickstart_ifcfg_get_identifier_names.erb

```diff
@@ -8,9 +8,13 @@ kind: snippet
 <%- if @identifier -%>
 <%=   "real=\"#{@identifier}\"" %>
 <%- else -%>
-<%=   "real=`grep -l #{@interface.inheriting_mac} /sys/class/net/*/{bonding_slave/perm_hwaddr,address} 2>/dev/null | awk -F '/' '// {print $5}' | head -1`" -%>
+<%-   if !@interface.inheriting_mac -%>
+<%=     "\nreal=`echo #{@interface.identifier}`" -%>
+<%-   else -%>
+<%=     "real=`grep -l #{@interface.inheriting_mac} /sys/class/net/*/{bonding_slave/perm_hwaddr,address} 2>/dev/null | awk -F '/' '// {print $5}' | head -1`" -%>
+<%-   end -%>
 <%-   if @interface.virtual? -%>
-<%=     "\nreal=`echo #{@interface.identifier} | sed s/#{@interface.attached_to}/$real/`" -%>
+<%=       "\nreal=`echo #{@interface.identifier} | sed s/#{@interface.attached_to}/$real/`" -%>
 <%-   end -%>
 <%- end -%>
 <%# ifcfg files are ignored by NM if their name contains colons so we convert colons to underscore %>
```

Comment:
Ok we must absolutely make sure that this does not introduce any regressions into the regular NIC provisioning. I can't tell from this honestly :-) I am going to test this patch.